### PR TITLE
IGAPP-718: Add map translations

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: translations/translations.json [hunk 1]
```diff
@@ -595,13 +567,13 @@
...
         "offer": "Dieses Angebot wurde leider nicht gefunden.",
-        "poi": "Dieser Point of Interest wurde leider nicht gefunden.",
+        "poi": "Dieser Ort wurde leider nicht gefunden.",
         "localNews": "Diese Nachricht wurde leider nicht gefunden.",
         "tunews": "Diese tünews wurde leider nicht gefunden.",
         "pageTitle": "Seite nicht gefunden"
       },
       "pageNotFound": "Diese Seite wurde leider nicht gefunden.",
-      "navigationApplication": "Keine Navigationsapplikation gefunden.",
+      "noSuitableAppAvailable": "Keine passende App verfügbar.",
```
Comment (klinzo) [inline translations/translations.json:576]: Not sure about this one, any suggestions?

**Comment by klinzo** [comment]
Please let me know if you think translations are missing/could be better. Also try to check that I did not break any keys/translations.

**Comment by klinzo** [comment]
I'd do the professional translations in this issue.

File: native/src/components/PoiDetails.tsx [hunk 2]
```diff
@@ -99,9 +99,7 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
         )}
         {distance && (
           <PoiDetailItem icon='place' language={language} onPress={detailPage ? navigateToPois : undefined}>
-            <Text>
-              {distance} {t('unit')} {t('distanceText')}
-            </Text>
+            <Text>{t('distanceKilometer', { distance })}</Text>
```
Comment (f1sh1918) [inline native/src/components/PoiDetails.tsx:102]:
```suggestion
            <Text>{t('distanceKilometre', { distance })}</Text>
```
would like to keep stuff en. but not important. if then pls also adjust translation key

File: native/src/components/PoiListItem.tsx [hunk 3]
```diff
@@ -54,17 +54,15 @@ type PoiListItemProps = {
...
     return (
       <StyledTouchableOpacity onPress={navigateToPoi} theme={theme} activeOpacity={1} language={language}>
         <Thumbnail source={thumbnail} />
         <Description theme={theme}>
           <Title theme={theme}>{poi.properties.title}</Title>
           {poi.properties.distance && (
-            <Distance theme={theme}>
-              {poi.properties.distance} {t('unit')} {t('distanceText')}
-            </Distance>
+            <Distance theme={theme}>{t('distanceKilometer', { distance: poi.properties.distance })}</Distance>
```
Comment (f1sh1918) [inline native/src/components/PoiListItem.tsx:65]: see comment above

File: native/src/routes/Pois.tsx [hunk 4]
```diff
@@ -194,11 +194,11 @@ const Pois = ({ pois, language, path, cityModel, navigateTo, navigateToFeedback,
           selectedFeature={selectedFeature}
           locationPermissionGranted={location !== null}
           onRequestLocationPermission={requestAndDetermineLocation}
-          fabPosition={sheetSnapPointIndex < snapPoints.length - 1 ? snapPoints[sheetSnapPointIndex]! : 0}
+          fabPosition={snapPoints[sheetSnapPointIndex] ?? 0}
```
Comment (f1sh1918) [inline native/src/routes/Pois.tsx:197]:
```suggestion
           fabPosition={sheetSnapPointIndex < snapPoints.length - 1 ? snapPoints[sheetSnapPointIndex]! : 0}
```

pls revert according to this issue

image

**Review by f1sh1918 — CHANGES_REQUESTED** [review]

**Comment by f1sh1918** [comment]
Tested on android couldn't find any translation issues so far! Good work!

File: native/src/routes/Pois.tsx (hunk 4 above)
Comment (klinzo) [reply native/src/routes/Pois.tsx:197]: Thx for spotting, I should really not do these kind of changes anymore :disappointed:

File: translations/translations.json [hunk 5]
```diff
@@ -5307,16 +5138,30 @@
...
-      "detailInformationHeader": "Information",
-      "detailAddressClipboardMessage": "Adresse wurde in die Zwischenablage kopiert"
+      "showOnMap": "Auf der Karte anzeigen",
+      "noPois": "Keine Orte gefunden.",
+      "unitKilometer": "km",
+      "unitMeter": "m",
+      "distanceKilometer": "{{distance}} km entfernt",
+      "distanceKilometerAlternative": "{{distance}} km",
+      "distanceMeter": "{{distance}} m entfernt",
+      "distanceMeterAlternative": "{{distance}} m",
```
[the remark this answers is not in the document]
Comment (klinzo) [reply translations/translations.json:5154]: We can remove anything we don't need once the feature is implemented. The reason why I did not just leave `entfernt` is that I can image the translation differs in some languages, e.g. the order of the words, which words are needed at all and so on. So imo it is not good to just combine some translated words together by ourselves.

**Review by Taggotty — APPROVED** [review]

**Review by f1sh1918 — APPROVED** [review]

**Comment by klinzo** [comment]
Translations have been exported and will be translated soonish. Once we get the results I'll merge this with the translations.

**Comment by klinzo** [comment]
Translations for malte only languages will be added in a separate PR as that may take a while. Please have another superficial look at the translations.json if it looks legit. It would be nice if you could translate some strings with deepl/google translate as a sample.

**Review by Taggotty — APPROVED** [review]

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**